pkg/controller/master/supportbundle/controller.go

```diff
@@ -0,0 +1,65 @@
+package supportbundle
+
+import (
+	"fmt"
+
+	harvesterv1 "github.com/harvester/harvester/pkg/apis/harvesterhci.io/v1beta1"
+	"github.com/harvester/harvester/pkg/generated/controllers/harvesterhci.io/v1beta1"
+	"github.com/harvester/harvester/pkg/settings"
```

Comment:
Conventionally we format the imports into three blocks, native/3rd parties/harvester packages.
In other words, the `goimports -local github.com/harvester/harvester` way.

Reply:
Re-format in https://github.com/harvester/harvester/pull/689/commits/e5cee0c74abbb1fbc385d67459c56fbf38fc80a0
I skip the generated codes.